packages/web-react/src/components/FileUploader/useFileUploaderStyleProps.ts

```diff
@@ -11,6 +12,14 @@ export interface FileUploaderStyleProps extends Validation {
   isDropZoneHidden?: boolean;
   isFluid?: boolean;
   queueLimitBehavior?: FileUploaderQueueLimitBehaviorType;
+  meta?: FileQueueValueMetaType;
+}
+
+interface ImageCropCSS extends CSSProperties {
```

Comment:
Can we define a dictionary like we do it with colors and use the keys here and also on line 84 where we are setting values?